mapswipe_workers/mapswipe_workers/firebase_to_postgres/update_data.py

```diff
@@ -88,42 +91,98 @@ def update_user_data(user_ids: list = []) -> None:
     logger.info("Updated user data in Potgres.")
 
 
-def update_project_data(project_ids: list = []):
+def get_project_attribute_from_firebase(project_ids: List[str], attribute: str):
+    """Use async for firebase sdk to query a project attribute.
+
+    Follows a workflow describted in this blogpost:
+    https://hiranya911.medium.com/firebase-python-admin-sdk-with-asyncio-d65f39463916
```

Comment:
A "simpler" approach here would be to not use the firebase admin sdk, but just rely on the REST api of firebase. For the two things for which we would use it (project status and username) this would even work without further authentication.

Maybe this would be then an easier approach where we could use the httpx module.

What do you think, should we explore this further?

Reply:
I adressed this in more detail in my comment of this file. Please have a look further down the page.